app/controllers/reminders/reminder/add-reminder.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    addReminder() {
+      console.log('woot there it is');
```

Comment:
Sent you both a DM but just documenting this in case Tay reads it - waiting on the file to be removed before merging